README.md

```diff
@@ -149,6 +149,7 @@ extension has the following configuration options:
 - `deno.codeLens.testArgs`: Provides additional arguments that should be set
   when invoking the Deno CLI test from a code lens. _array of strings, default
   `[ "--allow-all" ]`_.
+- `deno.codeLens.testEnvs`: Provides additional process environments
```

Comment:
These docs should be updated to provide more information about its purpose

Reply:
I've provided more information about its purpose.